README.md

```diff
@@ -12,33 +11,50 @@ git clone git@github.com:GoProperly/js-proper-logger.git
 npm install
 ```
 
+## Building
 
-## Running tests
+This project is build using TypeScript. Because of this there is a "build" step
+using rollup to generate a CommonJs and ESmodule bundle.
```

Comment:
```suggestion
This project is built using TypeScript. Because of this there is a "build" step
using Rollup to generate CommonJS and ESmodule bundles.
```